addons/advanced_fatigue/initSettings.inc.sqf

```diff
@@ -36,8 +39,9 @@
     [LSTRING(FadeStaminaBar), LSTRING(FadeStaminaBar_Description)],
     LSTRING(DisplayName),
     true,
-    false, {
-        if (!_this && GVAR(enabled) && GVAR(enableStaminaBar)) then {
+    0,
+    {
+        if (!_this && hasInterface && GVAR(enabled) && GVAR(enableStaminaBar)) then {
```

Comment:
are these hasInterface checks really necessary?

Reply:
Speed doesn't matter and no-interface works fine, just looks cleaner without it.